.github/workflows/npm-audit.yml

```diff
@@ -55,7 +55,6 @@ jobs:
         fail-fast: false
         matrix:
           library:
-            - msal-core
```

Comment:
We still need to run this in maintenance folder right? npm-audit identifies security bugs?

Reply:
It'll still be in the msal-lts branch. These audits run on PRs so we will still only see it when changes are made to core on the LTS branch.